### PR TITLE
Menus always appear above hovered code examples

### LINKED ISSUE #582: Hovering code block z-index higher than settings/help menus
## Bug Description

When settings or help menu are open, then hovering the code block it draws on top of the menu.
Hovering the code block this is relatively easy when the mouse is moving to one of the menu entries.

## How To Reproduce

1. Place a code block in a position on the screen as shown in the screenshot
2. Click on the settings
3. Mouse over the code block

## Expected behavior

Menu to be always on top

## Screenshots

Screenshot2022_02_14_095243

## Environment

**Desktop (please complete the following information):**
 - OS: macOS 11
 - Browser: Chrome
 - Version: v98

## Additional context

N/A

### PR DESCRIPTION
**Pull request by microbit-robert**
Prevents code pop overs from blocking open menus using z-index..

Closes #582.

### CODE REVIEW

File: src/settings/SettingsMenu.tsx [hunk 1]
```diff
@@ -54,7 +54,7 @@ const SettingsMenu = ({ size, ...props }: SettingsMenuProps) => {
           isRound
         />
         <Portal>
-          <MenuList>
+          <MenuList zIndex={3}>
```
Comment (microbit-matt-hillsdon) [inline src/settings/SettingsMenu.tsx:57]: Can we relate this to the other z-index? Perhaps add them alongside each other to zIndex.ts?

Comment (microbit-robert) [reply src/settings/SettingsMenu.tsx]: I added the z-index for the code popups to zIndex.ts and used the existing `zIndexAboveTerminal` for the menus. This futureproofs the zIndex for the menus when we get round to collapsing the sidebar [`ea44b60`](https://github.com/microbit-foundation/python-editor-next/pull/609/commits/ea44b60e1b6a8301b30743ec88de9d99fb1c3352).

Comment (microbit-matt-hillsdon) [reply src/settings/SettingsMenu.tsx]: Super, thanks.

**Review by microbit-matt-hillsdon — APPROVED** [review]
LGTM

**Pull request merged**